src/tests/ftest/container/autotest.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+
+
+class ContainerAutotestTest(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Tests container autotest.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize object."""
+        super().__init__(*args, **kwargs)
+
+    def test_container_autotest(self):
+        """Test container autotest.
+
+        :avocado: tags=all,full_regression,daily,hw,small,quick,autotest
+        """
+        self.pool = None
+        self.container = None
+        self.daos_cmd = None
```

Comment:
I mean you don't need these variables at all. You also don't need the `__init__()`.

Reply:
Code removed, in local run it shown no changes.

Reply:
Sorry I said you can make them locally above. It wasn't clear.